apps/dashboard/src/lib/components/Question/CheckboxQuestion/index.svelte

```diff
@@ -91,7 +93,7 @@
         </span>
       </svelte:fragment>
     </HtmlRender>
-    {#if !hideGrading}
+    {#if $course.type !== COURSE_TYPE.SELF_PACED && !hideGrading}
```

Comment:
@Chifez We shouldn't import or use `course` specific code into a reusable component.

You should rather make sure that we pass the right prop `hideGrading` and make it true if `$course.type === COURSE_TYPE.SELF_PACED`.